parlai/scripts/eval_ppl.py

```diff
@@ -180,7 +181,19 @@ def eval_ppl(opt, build_dict):
         but rather should have hardcoded settings for its dictionary.
```

Comment:
i dont understand the comment for "build_dict"

and...where is the comment  for dict_file ?